### PR TITLE
0.1.26

### PR DESCRIPTION
**Pull request by jonybur**
Fixing parsing for several 2021 packages, adding tests.

### CODE REVIEW

File: src/parsers/packets/CarStatusDataParser.ts [hunk 1]
```diff
@@ -13,7 +13,8 @@ export class CarStatusDataParser extends F1Parser {
         .floatle('m_fuelInTank')
         .floatle('m_fuelCapacity');
 
-    if (packetFormat === 2019 || packetFormat === 2020) {
+    if (packetFormat === 2019 || packetFormat === 2020 ||
+        packetFormat === 2021) {
```
Comment (Braendstrup) [inline src/parsers/packets/CarStatusDataParser.ts:17]: In javascript you can do multiple shorthand conditional checks by doing `packetFormat === ( 2019 || 2020 || 2021) `
Same functionality, less code

File: src/parsers/packets/CarStatusDataParser.ts [hunk 2]
```diff
@@ -22,41 +23,49 @@ export class CarStatusDataParser extends F1Parser {
...
 
-    this.uint8('m_engineDamage').uint8('m_gearBoxDamage');
+    if (packetFormat !== 2021) {
+      this.uint8('m_engineDamage').uint8('m_gearBoxDamage');
+    }
 
-    if (packetFormat === 2019 || packetFormat === 2020) {
-      this.int8('m_vehicleFiaFlags');
+    if (packetFormat === 2019 || packetFormat === 2020 ||
+        packetFormat === 2021) {
```
Comment (Braendstrup) [inline src/parsers/packets/CarStatusDataParser.ts:67]: Should be shortened like mentioned in earlier comment

File: src/parsers/packets/CarTelemetryDataParser.ts [hunk 3]
```diff
@@ -9,36 +9,46 @@ export class CarTelemetryDataParser extends F1Parser {
 
     if (packetFormat === 2018) {
       this.uint8('m_throttle').int8('m_steer').uint8('m_brake');
-    } else if (packetFormat === 2019 || packetFormat === 2020) {
+    } else if (
+        packetFormat === 2019 || packetFormat === 2020 ||
+        packetFormat === 2021) {
```
Comment (Braendstrup) [inline src/parsers/packets/CarTelemetryDataParser.ts:14]: Could be shortened

File: src/parsers/packets/PacketParticipantsDataParser.ts [hunk 4]
```diff
@@ -18,7 +18,8 @@ export class PacketParticipantsDataParser extends F1Parser {
       this.uint8('m_numCars');
     }
 
-    if (packetFormat === 2019 || packetFormat === 2020) {
+    if (packetFormat === 2019 || packetFormat === 2020 ||
+        packetFormat === 2021) {
```
Comment (Braendstrup) [inline src/parsers/packets/PacketParticipantsDataParser.ts:22]: Could be shortened

File: src/playground/index.ts [hunk 5]
```diff
@@ -1,20 +1,18 @@
...
 const socket = dgram.createSocket('udp4');
-socket.bind(5550);
+// socket.bind(5550);
 
 socket.on('message', (msg) => {
   const parsedmsg = F1TelemetryClient.parseBufferMessage(msg);
-  console.log(parsedmsg?.packetData?.data, parsedmsg?.packetID);
+  if (parsedmsg?.packetID === 'participants') {
+    // console.log(parsedmsg?.packetData?.data, parsedmsg?.packetID);
+  }
```
Comment (Braendstrup) [inline src/playground/index.ts:15]: Leftover debugging code?

File: src/playground/index.ts [hunk 6]
```diff
@@ -1,20 +1,18 @@
...
 const client = new F1TelemetryClient({
-  port: 20777,
-  forwardAddresses: [{port: 5550}],
-  skipParsing: true,
+  port: 20779,
 });
 
 const socket = dgram.createSocket('udp4');
-socket.bind(5550);
+// socket.bind(5550);
```
Comment (Braendstrup) [inline src/playground/index.ts:9]: Why commented?

**Review by Braendstrup — APPROVED** [review]

File: src/playground/index.ts (hunk 5 above)
Comment (jonybur) [reply src/playground/index.ts:15]: Yeah, still a wip

File: src/playground/index.ts (hunk 6 above)
Comment (jonybur) [reply src/playground/index.ts:9]: Just testing stuff (wip)

**Pull request merged**